process_manager/views/partials.py

```diff
@@ -62,13 +61,8 @@ def process_table(request: HttpRequest) -> HttpResponse:
 
 @login_required
 def messages(request: HttpRequest) -> HttpResponse:
-    """Renders and pops Kafka messages from the user's session."""
-    with transaction.atomic():
-        # atomic to avoid race condition with kafka consumer
-        messages = request.session.load().get("messages", [])
-        request.session.pop("messages", [])
-        request.session.save()
-
+    """Renders Kafka messages from the database."""
+    messages = [f"{msg.timestamp}: {msg.message}" for msg in DruncMessage.objects.all()]
```

Comment:
In theory, we should be pulling the messages only relevant for this user, which means that there should be user-related information in the `DruncMessage` model, right?

Reply:
That was my initial assumption but after checking with the project team all users need to see all messages so there isn't a requirement for this.

Reply:
Odd, but so be it.